src/AppBundle/Controller/InterviewController.php

```diff
@@ -408,4 +408,21 @@ public function cancelByResponseCodeAction(Request $request, Interview $intervie
             'form' => $form->createView(),
         ));
     }
+
+    /**
+     * @param Request $request
+     * @param Interview $interview
+     *
+     * @return \Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function editStatusAction(Request $request, Interview $interview)
+    {
+        $status = $request->get('status');
```

Comment:
Her er `$status` en string. Interview forventer å få en status som Int. PHP kommer nok til å prøve å gjøre en konvertering automatisk, men det er nok uansett lurt å slenge en `intval()` rundt her. Så blir det også litt klarere for oss som skal lese at dette er en int.